.gitignore

```diff
@@ -1,2 +1,4 @@
 *.o
-os-update-minui
+RPMS
+yamui
+screensaverd
```

Comment:
This will clash in the future with the systemd built in screensaverd when it comes ;)